build.rs

```diff
@@ -246,14 +246,6 @@ fn build_mln() {
         }
     }
 
-    println!("cargo:rustc-link-lib=sqlite3");
-    println!("cargo:rustc-link-lib=uv");
-    println!("cargo:rustc-link-lib=icuuc");
-    println!("cargo:rustc-link-lib=icui18n");
-    //println!("cargo:rustc-link-lib=nu"); // todo add to docs => git clone https://bitbucket.org/alekseyt/nunicode.git && cmake .  && make && sudo make install
-    println!("cargo:rustc-link-lib=jpeg");
-    println!("cargo:rustc-link-lib=png");
-    println!("cargo:rustc-link-lib=webp");
     println!("cargo:rustc-link-lib=curl");
     println!("cargo:rustc-link-lib=z");
```

Comment:
On slack you hinted that we might not need a few of those linking dependencys below.
Is this a different PR? 🤔